packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -85,106 +88,205 @@ type LoadingTileProps = {
   description: boolean;
   hasTopSection: boolean;
   image: boolean;
-  overline: boolean;
   title: boolean;
+  variant: AppTileProps["variant"];
 };
 
 const ImageContainer = styled("div", {
-  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" && prop !== "variant",
 })<{
   odysseyDesignTokens: DesignTokens;
-}>(({ odysseyDesignTokens }) => ({
+  variant: AppTileProps["variant"];
+}>(({ odysseyDesignTokens, variant }) => ({
   display: "flex",
   alignItems: "flex-start",
-  maxHeight: APP_TILE_IMAGE_HEIGHT,
-  marginBlockEnd: odysseyDesignTokens.Spacing5,
+  justifyContent: variant === "compact" ? "center" : "flex-start",
+  maxHeight:
+    variant === "compact"
+      ? APP_TILE_COMPACT_IMAGE_HEIGHT
+      : APP_TILE_IMAGE_HEIGHT,
+  marginBlockEnd:
+    variant === "compact"
+      ? odysseyDesignTokens.Spacing4
+      : odysseyDesignTokens.Spacing3,
 
   ["img"]: {
-    height: APP_TILE_IMAGE_HEIGHT,
+    height:
+      variant === "compact"
+        ? APP_TILE_COMPACT_IMAGE_HEIGHT
+        : APP_TILE_IMAGE_HEIGHT,
+    width: variant === "compact" ? "100%" : "auto",
```

Comment:
If we are setting a `height` on these images, we probably should not set a `width`. It can skew the images

Reply:
What happens in the event someone uses an img that is much wider than it is tall? Will it overflow the container?

Reply:
This is how it looks in practice, with multiple aspect ratios:

<img width="1049" alt="Screenshot 2024-11-20 at 3 00 24 PM" src="https://github.com/user-attachments/assets/92a8ea87-4c39-4fee-9cdb-cf0c9479f19d">

Reply:
It's hard to tell with these types of placeholder images. Have you tried it with more realistic images? My hunch is we get some interesting results. Mainly that the image would be very skewed in the 'wide' case

Reply:
But, I could be wrong.

Reply:
<img width="643" alt="Screenshot 2024-11-20 at 4 49 29 PM" src="https://github.com/user-attachments/assets/57f41125-e2e7-453c-b433-dc28eaef39a9">

I was able to manually verify that each image maintains the aspect ratio of its full-size equivalent